definitions/variable/variables.yaml

```diff
@@ -706,6 +706,40 @@
 - Emissions|Kyoto Gases|Fugitive Emissions:
     definition: Kyoto GHG emissions from fugitive emissions (IPCC category 1B)
     unit: Mt CO2-equiv/yr
+- Emissions|Kyoto Gases|Energy|Supply:
+    definition: 'Kyoto GHG emissions from fuel combustion and fugitive emissions from
+      fuels: electricity and heat production and distribution (IPCC category 1A1a),
+      other energy conversion (e.g. refineries, synfuel production, solid fuel processing,
+      IPCC category 1Ab, 1Ac), incl. pipeline transportation (IPCC category 1A3ei),
+      fugitive emissions from fuels (IPCC category 1B) and emissions from carbon dioxide
+      transport and storage (IPCC category 1C), net of negative emissions from Bioenergy
+      with CCS (BECCS), exc. industrial autoproduction'
+    unit: Mt CO2-equiv/yr
+- Emissions|Gross|Kyoto Gases|Energy|Supply:
```

Comment:
Was this agreed with Elmar Kriegler and Jessica Strefler? It seems that there is still an ongoing discussion in the NAVIGATE project, with a slight preference towards "Gross Emissions|..."

Reply:
This was not agreed with Elmar Kriegler and Jessica Strefler. For CO2 emissions btw, we have the formulation "Gross Fossil CO2" at the moment in ARIADNE. It would be important to have these types of variables to report and plot correctly aggregated REMIND emissions. From my view, the harmonization with NAVIGATE is not really necessary here because it will slow down the process and there are probably other parts of the ARIADNE variable template which are not in line with the NAVIGATE template either. However, if there was a general agreement by the project leaders to not add further ARIADNE variables which are not in NAVIGATE, then we need to talk in a larger group.

Reply:
We decided to talk to Gunnar about this before we proceed to be on the safe side.

Reply:
@danielhuppmann Gunnar says, we should go ahead. Synching with NAVIGATE is nice-to-have, but not worth the effort in this case. We intend to do this later.

So if there are no other concerns, please go ahead and merge.